lib/engine/config/game/g_1856.rb

```diff
@@ -517,18 +517,24 @@ module G1856
       "sym": "CGR",
       "logo": "1856/CGR",
       "name": "Canadian Government Railway",
-      "tokens": [
-        100,
-        100,
-        100,
-        100,
-        100,
-        100,
-        100,
-        100,
-        100
-      ],
-      "color": "cgrBlack"
+      "tokens": [],
+      "color": "cgrBlack",
+      "abilities": [
+         {
+            "type": "train_buy",
+            "description": "Inter train buy/sell at face value",
+            "face_value": true
+         },
+         {
+            "type": "train_limit",
+            "description": "+1 train limit",
+            "increase": 100
+         },
+         {
+            "type": "no_buy",
+            "description": "Unavailable until nationalization"
```

Comment:
I think we might beed to general work on this ability. Maybe it should be part of the general code to check for this no_buy (in can_par, can_buy, can_exchange) and I think it is not today.
Also we could consider having some timing attribute, when it should stop?

Is used for NdM in 18MEX and I am using it in the upcoming 1824.

Reply:
Right now this ability is only there as a visual aid, and it is controlled by the 1856::Corporation class. I think we could make it into an actual ability where having the ability makes it unparrable until the ability is removed